### ISSUE TITLE
Shrink CSS and JS on deployment

### ISSUE THREAD

**Issue by cultcom** [debops-contrib/ansible-roundcube#17]
bin/cssshrink.sh and bin/jsshrink.sh should be called after deployment to generate *.min.* files.

unfortunately they need a java runtime environment. Maybe it is a better idea to forget about git and deploy roundcube by the official upstream archives.

**Comment by ganto** [comment]
You are right, this is not easy doable when doing the deployment via git. In my PR regarding the upgrade to Roundcube 1.3, I didn't change the deployment method as I worried, this would require a too big change (more or less a complete rewrite).

With the experience of all the stuff that I added and fixed in #24, I guess changing the deployment method to upstream archives still should be considered for the future. It would simplify especially the PHP composer bits and the entire post-deployment tasks including the one mentioned in this issue. I therefore leave it open and unresolved until someone want to implement this.
